src/stylesheets/components/_site-search.scss

```diff
@@ -0,0 +1,186 @@
+.app-site-search {
+  float: left;
+  margin-bottom: govuk-spacing(2);
+  margin-top: govuk-spacing(2);
+  max-width: calc(100% - 120px);
+  width: 100%;
+
+  @include govuk-media-query($from: tablet) {  
+    max-width: 100%;
+    width: 300px;
+    float: none;
+  }
+
+  @include govuk-media-query($from: desktop) {
+    float: right;
+    margin: 0;
+    margin-top: -6px;
```

Comment:
@hannalaakso @igloosi The -6px is to vertically align the site search in the header. The header itself needs to retain it's height but because of the internal padding is actually too small to fit the search bar in.

I'd suggest the comment should just be "negative margin to vertically align search in header"

<img width="307" alt="screen shot 2018-08-14 at 08 34 10" src="https://user-images.githubusercontent.com/23356842/44077875-f0aee1a8-9f9c-11e8-8901-6d680c4addd2.png">

Reply:
added comment